lib/utils/timetable.ts

```diff
@@ -116,26 +117,41 @@ export const toDefaultLocalDateString = (dateString: string) => {
   });
 };
 
-type StageProgress = "IN PROGRESS" | "FINISHED" | "NOT STARTED";
+const getDatePartFromISOString = (isoString: string, includeDays: boolean) => {
+  const date = isoString.split("T")[0];
+  return includeDays ? date : date.split("-").slice(0, 2).join("-");
+};
 
 export const getStageProgress = (
   lastUpdatedDate: string,
   startEventDate: string,
   endEventDate?: string
-): StageProgress => {
-  const startDate = new Date(startEventDate);
-  const referenceDate = new Date(lastUpdatedDate);
+): TagLabel => {
+  const includeDays = startEventDate.split("-").length > 2;
```

Comment:
Maybe not the nicest place or best way to do this but it looks like we're thinking of making the dates mandatory, so we should be safe to assume both dates are the same format